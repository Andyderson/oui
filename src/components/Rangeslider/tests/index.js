import React from 'react';
import RangeSlider from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

describe('components/Toggle', () => {
  let component;

  beforeEach(() => {
    component = shallow(
      <RangeSlider />
    );
  });

  it('should render component same as snapshot', () => {
    expect(shallowToJson(component)).toMatchSnapshot();
  });

  it('should render component with correct props', () => {
    expect(component.length).toBe(1);
  });

});
