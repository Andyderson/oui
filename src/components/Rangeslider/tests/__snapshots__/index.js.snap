// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Toggle should render component same as snapshot 1`] = `
<div
  className="oui-rangeslider"
>
  <div
    className="oui-grid"
  >
    <div
      className="oui-grid__cell position--relative"
    >
      <div
        className="flex"
      >
        <label
          className="oui-label muted flush flex--1"
        >
          0%
        </label>
        <label
          className="oui-label muted flush"
        >
          100%
        </label>
      </div>
      <div
        className="range-display"
      >
        <div
          className="range-display range-display-overlay"
          style={
            Object {
              "left": "undefined%",
            }
          }
        />
      </div>
      <div
        className="range-grid"
      >
        <div
          className="gridline grid-1"
        />
        <div
          className="gridline grid-2"
        />
        <div
          className="gridline grid-3"
        />
        <div
          className="gridline grid-4"
        />
        <div
          className="gridline grid-5"
        />
        <div
          className="gridline gridline-half grid-h1"
        />
        <div
          className="gridline gridline-half grid-h2"
        />
        <div
          className="gridline gridline-half grid-h3"
        />
        <div
          className="gridline gridline-half grid-h4"
        />
      </div>
      <input
        className="hard position--relative width--1-1"
        data-traffic-group="1"
        max="100"
        min="0"
        step="1"
        type="range"
      />
    </div>
  </div>
</div>
`;
